simplq/src/styles/homePage.module.scss

```diff
@@ -143,3 +143,110 @@
     font-size: small;
   }
 }
+
+.burger{
+  width: 2rem;
+  height: 2rem;
+  position: fixed;
+  top: .5rem;
+  right: 1rem;
+  z-index: 5;
+  display: none;
+  @media (max-width: 768px) {
+    display: flex;
+    justify-content: space-around;
+    flex-flow: column nowrap;
+  }
+  div {
+    width: 2rem;
+    height: 0.25rem;
+    $type: open;
+    @if $type == open {
+      background-color: #fff;
+    } @else{
+      background-color: #3a3768;
+    }
+    border-radius: .5rem;
+    transform-origin: 1px;
+    transition: all 0.3s linear;
+  
+    &:nth-child(1) {
+      $type: open;
+      @if $type == open {
+        transform: rotate(45deg);
+      } @else{
+          transform: rotate(0);
+      }
+    }
+    &:nth-child(2) {
+      $type: open;
+      @if $type == open {
+        transform: translateX(100%);
+        opacity: 0;
+      } @else{
+          transform: translateX(0);
+          opacity: 1;
+      }
+    }
+    &:nth-child(3) {
+      $type: open;
+      @if $type == open {
+        transform: rotate(-45deg);
+      } @else{
+          transform: rotate(0);
+      }
+    }
+  }
+}
+
+  .menu-list{
+    list-style: none;
+    display: flex;
+    flex-flow: row nowrap;
+    margin: 0;
+    position: fixed;
+    top: 1.2rem;
+    li {
+      color: #fff;
+      padding: 0 2rem;
+    }
```

Comment:
let's reduce the padding of li to `padding: 0 1rem;`

Then, for the menu-list, to adjust for the image (which has a total width of `4rem`), we would have to give `left: 3rem` (4/2 for distance from image's center, and 1 rem for a margin)

So finally this will look like 
```
  .menu-list{
    top: 1.2rem;
    left: 3rem;
    li {
      ...
      padding: 0 1rem;
    }
}
```

You can piggyback this in your next PR.

Reply:
Okay, I will do these changes.